src/views/components/survey_components/question_time_setting.vue

```diff
@@ -0,0 +1,368 @@
+                    <div v-if="contain_up_arrow(question)" v-on:click="up_exchange(question)">&uarr;</div>
+                    <!-- The 'move down' arrow of the question -->
+                    <div v-if="contain_down_arrow(question)" v-on:click="down_exchange(question)">&darr;</div>
+                </li>
+            </center></ol>
+        </div>
+        <!-- Operations on the selected questions -->
+        <div id="change_question" v-if="edit_click">
+            <br/>
+            <br/>
+            <br/>
+            <br/>
+            Change Question choice from &ensp;
+            <!-- The original selected questions -->
+            <select v-model="selected">
+                <option v-for="question in questions">{{question.content}}</option>
+            </select>
+            &ensp;to &ensp;
+            <!-- The back up questions to choose from -->
+            <select v-model="unselected">
+                <option v-for="question in backup_questions">{{question.content}}</option>
+            </select>
+            <br/>
+            <br/>
+            <br/>
+            <button type="submit" v-on:click="replace(selected, unselected)">
+                Change</button>
+            &ensp;
+            <button type="submit" v-on:click="cancel">Cancel</button>
+            </br>
+        </div>
+        <br/>
+
+        <h2><b>Set the start time</b></h2>
+        <p>(Note: Please use the same format as default)</p>
+        <input type="datetime-local" v-model="start_time"/>
+        <h2><b>Set the end time</b></h2>
+        <p>(Note: Please use the same format as default)</p>
+        <input type="datetime-local" v-model="end_time"/>
+        <br/><br/><br/><br/>
+
+        <button type="submit" v-on:click="finish">
+            Finish the setting</button>
+    </div>
+</template>
+
+<script>
+export default {
+    name: "question_time_setting",
+    data: function () {
+        return {
+            survey_package: null,
+            num_locked: 0,
+            selected: null,
+            unselected: null,
+            survey_id: null,
+            name: null,
+            edit_click: false,
+            checkbox_indices: [],
+            check_status: [0, 0],
+            term: "2018Winter",
+            start_time: "2000-01-01T00:00:00",
+            end_time: "2000-01-01T00:00:00",
+            // The questions is gonna be a list of question object
+            questions: null,
+            // The backup questions is gonna be a list of back up questions
+            backup_questions: []
+        }
+    },
+    created: async function () {
+        this.survey_id = this.$route.query.survey_id;
+        this.name = this.$route.query.name;
+        // Fetch the list of questions from the API call
+        let fetched = await fetch(
+            "http://localhost:3000/get_info.php?what=surveys&user_id=admin0&survey_id=1"
+        );
+        let fetchedJSON = await fetched.json();
+        console.log(fetchedJSON);
```

Comment:
Shouldn't have `console.log` statements in your final code.

Reply:
Sure